benchmark/progress-bar.js

```diff
@@ -0,0 +1,35 @@
+const cliProgress = require('cli-progress');
+
+exports.createProgressBarFromBenchmark = benchmark => {
+    const progressBar = new cliProgress.SingleBar({
+        format: ' [{bar}] | {value} / {total} | {status}',
+        stopOnComplete: true,
+        forceRedraw: true,
+        autopadding: true,
+        clearOnComplete: true
+    });
+
+    const benchmarkTasksCount = benchmark.tasks.length + 1; // warmup;
+    let doneTasksCount = 0;
+
+    benchmark.addEventListener('warmup', () => {
+        progressBar.start(benchmarkTasksCount, doneTasksCount, {status: 'Warming up the benchmark'});
+    });
+
+    benchmark.addEventListener('start', () => {
+        doneTasksCount++;
+    });
```

Comment:
"start" is emitted on "warmup" end. Progress bar need to count "warmup" as a task too